kv/local_sender_test.go

```diff
@@ -43,6 +44,36 @@ func TestLocalSenderAddStore(t *testing.T) {
 	}
 }
 
+func TestLocalSenderRemoveStore(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	ls := NewLocalSender()
+
+	store := storage.Store{}
+	ls.AddStore(&store)
+
+	ls.RemoveStore(&store)
+
+	if ls.HasStore(store.Ident.StoreID) {
+		t.Errorf("expted local sender to remove storeID=%d", store.Ident.StoreID)
+	}
+}
+
+func TestLocalSenderGetStoreIDs(t *testing.T) {
```

Comment:
`GetStoreIDs` is not used anywhere and I don't expect us to use it in the future. We should just remove it.